src/library/ExtendedSplashScreen.Uno.WinUI/ExtendedSplashScreen.Uno.WinUI.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
   <PropertyGroup>
-    <TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-ios;net6.0-macos;net6.0-maccatalyst;net6.0-android</TargetFrameworks>
```

Comment:
Do you need to remove those targets?

Reply:
I removed them because the solution wasn't working at all in Visual Studio. I tried cleaning some stuff and doing this fixed the issue. Also I don't even know if it works on Mac at all and why we would need to target it. Unnecessary noise in my opinion.

Reply:
I'm currently adding them back. Still make the solution unusable on .NET 7 at least. Will see if CI is alright.

Reply:
I think it broke CI, removing it.